### PR TITLE
DHIS2-11045_WarningWidget

### PR DESCRIPTION
**Pull request by eirikhaugstulen**
Added documentation on the Warning widget in the enrollment overview.

### CODE REVIEW

**Review by karolinelien — APPROVED** [review]
LGTM 👍

**Review by simonadomnisoru — APPROVED** [review]

File: src/user/using-the-capture-app.md [hunk 1]
```diff
@@ -872,6 +872,14 @@ On the enrollment dashboard, you can view the Person Profile widget.
 
 The Person profile widget is used for viewing key attributes but not for editing. In order to edit the person profile, you must select the `Edit` button to open an edit window. 
 
+### Warning widget
+On the enrollment dashboard, the warning widget will be displayed.
+
+![](resources/images/capture_app/enrollment-dash-warning-widget-1.png)
+
+The warning widget will display warnings recieved from configured program rules which are not associated with a specific data element or attribute.
```
Comment (j-nguyenn) [inline src/user/using-the-capture-app.md]: NIT: typo `received`

File: src/user/using-the-capture-app.md [hunk 2]
```diff
@@ -872,6 +872,14 @@ On the enrollment dashboard, you can view the Person Profile widget.
 
 The Person profile widget is used for viewing key attributes but not for editing. In order to edit the person profile, you must select the `Edit` button to open an edit window. 
 
+### Warning widget
+On the enrollment dashboard, the warning widget will be displayed.
```
Comment (cooper-joe) [inline src/user/using-the-capture-app.md]: I suggest removing this line and incorporating it into the sentence below, see next suggestion.

File: src/user/using-the-capture-app.md (hunk 1 above)
Comment (cooper-joe) [inline src/user/using-the-capture-app.md]: I suggest simplifying and adding in the previous line:
`On the enrollment dashboard, the warning widget displays warnings related to the current dashboard. The widget shows warnings that are not associated with any specific data item.`

This is user documentation, so we should, as I understand it, avoid referring to the DHIS2 data model.

File: src/user/using-the-capture-app.md [hunk 3]
```diff
@@ -872,6 +872,14 @@ On the enrollment dashboard, you can view the Person Profile widget.
...
+
+![](resources/images/capture_app/enrollment-dash-warning-widget-1.png)
+
+The warning widget will display warnings recieved from configured program rules which are not associated with a specific data element or attribute.
+If there are no warnings, the warning widget will not be displayed.
```
Comment (cooper-joe) [inline src/user/using-the-capture-app.md]: I suggest rewording:
`If there aren't any warnings to show for the current dashboard then the widget is hidden.`

**Review by cooper-joe — CHANGES_REQUESTED** [review]

**Comment by eirikhaugstulen** [comment]
Thanks!
Implemented the changes!

**Review by cooper-joe — APPROVED** [review]
